Include <sys/time.h> for the gettimeofday function.  This prevents an
implicit function declaration and build failures with future
compilers.

Submitted upstream: <https://sourceforge.net/p/iaxclient/patches/57/>

diff -ur iaxclient-2.1beta3.orig/simpleclient/stresstest/file.c iaxclient-2.1beta3/simpleclient/stresstest/file.c
--- iaxclient-2.1beta3.orig/simpleclient/stresstest/file.c	2008-04-07 18:05:27.000000000 +0200
+++ iaxclient-2.1beta3/simpleclient/stresstest/file.c	2023-02-24 15:00:47.282858961 +0100
@@ -1,6 +1,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <stdio.h>
+#include <sys/time.h>
 #include "file.h"
 
 #ifdef __GNUC__
diff -ur iaxclient-2.1beta3.orig/simpleclient/stresstest/stresstest.c iaxclient-2.1beta3/simpleclient/stresstest/stresstest.c
--- iaxclient-2.1beta3.orig/simpleclient/stresstest/stresstest.c	2008-04-07 18:05:27.000000000 +0200
+++ iaxclient-2.1beta3/simpleclient/stresstest/stresstest.c	2023-02-24 15:00:33.952996082 +0100
@@ -24,6 +24,7 @@
 #include <stdio.h>
 #include <time.h>
 #include <signal.h>
+#include <sys/time.h>
 
 #include "iaxclient.h"
 #include "file.h"
